mm2src/coins/utxo/utxo_builder/utxo_arc_builder.rs

```diff
@@ -17,6 +17,7 @@ use spv_validation::helpers_validation::validate_headers;
 use spv_validation::storage::BlockHeaderStorageOps;
 
 const BLOCK_HEADERS_LOOP_INTERVAL: f64 = 60.;
+const BLOCK_HEADERS_LOOP_SLEEP_TIMER: f64 = 10.;
```

Comment:
You can rename this `BLOCK_HEADERS_LOOP_ERORR_SLEEP_TIMER` and use it here too instead of `BLOCK_HEADERS_LOOP_INTERVAL` https://github.com/KomodoPlatform/atomicDEX-API/blob/a8df84d660399e1babd6cb4df5b0f9435cbb39ba/mm2src/coins/utxo/utxo_builder/utxo_arc_builder.rs#L331